AbletonMCP_Remote_Script/AbletonMCP_Remote_Script/handlers/clips.py

```
@@ -0,0 +1,766 @@
+            import Live
+            if hasattr(Live.Clip, 'MidiNoteSpecification'):
+                specs = []
+                for s in note_specs:
+                    specs.append(Live.Clip.MidiNoteSpecification(
+                        pitch=s["pitch"], start_time=s["start_time"],
+                        duration=s["duration"], velocity=s["velocity"],
+                        mute=s["mute"]))
+                clip.add_new_notes(tuple(specs))
+                return {"note_count": len(notes)}
+        except Exception:
+            pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)